pkg/backup/backup.go

```diff
@@ -46,25 +45,30 @@ const (
 	maxRecentBackupStatusCount = 10
 )
 
-type Backup struct {
+// BackupManager manages how to backup.
```

Comment:
what does how mean? need to be more descriptive.

Reply:
`BackupManager manages where to get etcd backups from and where save them`?